src/scrabble.js

```diff
@@ -1,6 +1,75 @@
+const letterArray = [
+  "A",
+  "E",
+  "I",
+  "O",
+  "U",
+  "L",
+  "N",
+  "R",
+  "S",
+  "T",
+  "D",
+  "G",
+  "B",
+  "C",
+  "M",
+  "P",
+  "F",
+  "H",
+  "V",
+  "W",
+  "Y",
+  "K",
+  "J",
+  "X",
+  "Q",
+  "Z",
+];
+const valueArray = [
+  1, 1, 1, 1, 1, 1, 1, 1, 1, 1, 2, 2, 3, 3, 3, 3, 4, 4, 4, 4, 4, 5, 8, 8, 10,
```

Comment:
A better way to do this, which would make it more maintainable, would be to put both the letters and corresponding values as key value pairs in an object, e.g. 
`const scoresObject = {
 A: 1,
  E: 1,
  ...etc
}`